src/docs/demos/modal/modal-demo.controller.js

```diff
@@ -1,13 +1,9 @@
 class controller {
-  constructor ($ngbsModal) {
-    this.$ngbsModal = $ngbsModal;
+  constructor($log, $ngbsModal) {
+    Object.assign(this, { $log, $ngbsModal });
```

Comment:
Ahh, I took it granted, thinking your Webpack config already supported `Object.assign`!

As for plugin vs polyfill, it always seems best to let Babel handle it to reduce the need for extra code on the client.